### PR TITLE
Sharing alerts

### PR DESCRIPTION
**Pull request by rajsarkapally-sfdc**

### CODE REVIEW

File: ArgusWebServices/src/main/java/com/salesforce/dva/argus/ws/resources/AlertResources.java [hunk 1]
```diff
@@ -69,986 +69,991 @@
...
+	 * @return  Updated notification object.
+	 *
+	 * @throws  WebApplicationException  The exception with 404 status will be thrown if either an alert or notification do not exist.
+	 */
+	@PUT
+	@Produces(MediaType.APPLICATION_JSON)
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/{alertId}/notifications/{notificationId}")
+	@Description("Updates a notification having the given notification ID if associated with the given alert ID.")
+	public NotificationDto updateNotificationById(@Context HttpServletReq
```
Comment (bsura) [inline ArgusWebServices/src/main/java/com/salesforce/dva/argus/ws/resources/AlertResources.java]: @rajsarkapally-sfdc Looks like there are no changes for AlertResources file. Is that the case?

Comment (rajsarkapally-sfdc) [inline ArgusWebServices/src/main/java/com/salesforce/dva/argus/ws/resources/AlertResources.java]: added just following one line at line number 100

result.addAll(alertService.findSharedAlerts());

File: ArgusCore/src/main/java/com/salesforce/dva/argus/entity/Alert.java [hunk 2]
```diff
@@ -82,473 +82,518 @@
...
+		}
+		return true;
+	}
+
+	@Override
+	public String toString() {
+		return "Alert{" + "name=" + name + ", expression=" + expression + ", cronEntry=" + cronEntry + ", enabled=" + enabled +
+				", missingDataNotificationEnabled=" + missingDataNotificationEnabled + ", notifications=" + notifications + ", triggers=" + triggers +
+				", owner=" + owner + '}';
+	}
```
Comment (bsura) [inline ArgusCore/src/main/java/com/salesforce/dva/argus/entity/Alert.java]: You might want to add shared to toString()

**Comment by bsura** [comment]
@rajsarkapally-sfdc I added one more comment. Also this PR is out-of-date with the base branch. Can you manually update it, sign the CLA and re-submit the PR.

**Pull request closed without merging**